packages/chain/contracts/Eth_broker.sol

```diff
@@ -22,7 +22,6 @@ contract Eth_broker {
     // -------------------------------------------------------------------------
     // Events
     // -------------------------------------------------------------------------
-
```

Comment:
I know it is not part of this changeset, but I notice that you are using Solidity version 0.5.0 which is rather outdated. Is it in the books to update this?